test/e2e/storage/testsuites/volume_io.go

```diff
@@ -0,0 +1,370 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
A general question: have you timed runs of these tests vs. the old ones? are there any significant performance increases/decreases? Are we running the exact same set of tests (at least the same number) now as before

Reply:
This refactoring will run more tests than before.  Which is good.  Previously, you needed to create a test case for every plugin.  This refactoring makes it easier to run any test case against any plugin.